### ISSUE TITLE
Will fastcgo support general c function call？

### ISSUE THREAD

**Issue by loadwiki** [petermattis/fastcgo#1]

**Comment by loadwiki** [comment]
Fastcgo can call c function with 0-4 zero unint64 argument with no return value. Will it support general c function call, such as return value, mutli-type input argument,variable number of arguments and so on.

**Comment by petermattis** [comment]
It will if you supply a patch. I'm not sure how that could be achieved, though.

**Comment by loadwiki** [comment]
Sorry I have little experience with assembly . It should be a hard work. Does the rustgo support general function call?

**Comment by petermattis** [comment]
No, rustgo doesn't support general calls (nor is it even available as far as I know) unless you write some custom assembly for each call.
